script/test.sh

```diff
@@ -39,9 +39,10 @@ teardown () {
 test_check () {
   NUM=$1
   EXPECTED_ADDED_FILE_NUM=$2
-  git log --oneline
-  ./$ExecComamnd -n $NUM -f -d
-  git log --oneline
+  OPTIONS=$3
+  #git log --oneline
```

Comment:
is not this needed in any case?
if it is so, please remove.
if it is NOT, please issue the tasks.

Reply:
OPTION value is needed to test this feature.

This test case shows debug log. OPTION value is set "-d"
```
*** START 0..5 ***
+ test_check 0..5 6 -d
+ NUM=0..5
+ EXPECTED_ADDED_FILE_NUM=6
+ OPTIONS=-d
+ ./qs -n 0..5 -f -d
INFO[0000] *** force update ***
INFO[0000] git rebaes HEAD~5 --exec=git commit --amend -m"Add file-5"
```

This test case suppresses debug log. OPTION value is set none.
```
*** START 0..5 ***
+ test_check 0..5 6
+ NUM=0..5
+ EXPECTED_ADDED_FILE_NUM=6
+ OPTIONS=
+ ./qs -n 0..5 -f
Successfully rebased and updated refs/heads/master.
```

Reply:
I know that the OPTION variable is necessary,
I pointed out `#git log --oneline`
Sorry to misunderstand

Reply:
I remove the lines.